ring-jetty-adapter/test/ring/adapter/test/jetty.clj

```diff
@@ -284,13 +284,28 @@
                                 :send-server-version? false}
         (let [response (http/get test-url)]
           (is (not (contains? (:headers response) "Server")))))))
-
   (testing "excluding cipher suites"
     (let [cipher  "SSL_RSA_WITH_NULL_MD5"
           options (assoc test-ssl-options :exclude-ciphers [cipher])
           server  (run-jetty echo-handler options)]
       (try
         (is (contains? (exclude-ciphers server) cipher))
+        ;; The operation is additive; it doesn't replace ciphers that Jetty
+        ;; excludes by default
+        (is (> (count (exclude-ciphers server)) 1))
```

Comment:
Is this an adequate extension of the test do you think? Or something like a 'before vs after' would be more worthwhile? Or perhaps this is even an unncessary addition

Reply:
Yes, I think this is fine as a quick test.